velox/common/memory/HashStringAllocator.cpp

```diff
@@ -83,6 +83,10 @@ std::string HashStringAllocator::Header::toString() {
 
 HashStringAllocator::~HashStringAllocator() {
   clear();
+#ifdef NDEBUG
+  VELOX_CHECK_EQ(state_.cumulativeBytes(), 0);
```

Comment:
You can use VELOX_DCHECK_EQ here?

Reply:
Can you move these two checks in clear()?

Reply:
I've moved it in clear() as suggested. Thanks.